### PR TITLE
chore: Add governance params

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
People would like to have the NNS parameters in the aggregator.

# Changes
- Add the NNS parameters.  Simple pass-through, nothing special.

# Tests
- See: https://otgyv-wyaaa-aaaak-qcgba-cai.icp0.io/

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: rs/sns_aggregator/src/types/slow.rs [hunk 1]
```diff
@@ -47,21 +49,38 @@ pub struct SlowSnsData {
 
 impl From<&UpstreamData> for SlowSnsData {
     fn from(upstream: &UpstreamData) -> Self {
+        let UpstreamData {
```
Comment (bitdivine) [inline rs/sns_aggregator/src/types/slow.rs:52]: Note: Unpacking makes it hard to forget to add a new upstream field to the slow data.  The new field must be added or explicitly ignored.

File: rs/sns_aggregator/src/upstream.rs [hunk 2]
```diff
@@ -167,6 +168,12 @@ async fn get_sns_data(index: u64, sns_canister_ids: DeployedSns) -> anyhow::Resu
         .map_err(|err| crate::state::log(format!("Failed to get lifecycle: {err:?}")))
         .ok();
 
+    let nervous_system_parameters: Option<NervousSystemParameters> =
+        get_nervous_system_parameters(governance_canister_id)
+            .await
+            .map_err(|err| crate::state::log(format!("Failed to get nervous system parameters: {err:?}")))
+            .ok();
```
Comment (bitdivine) [inline rs/sns_aggregator/src/upstream.rs:175]:
```
1702391662381769661: Failed to get nervous system parameters: (CanisterError, "IC0503: Canister tr3th-kiaaa-aaaaq-aab6q-cai trapped explicitly: Panicked at 'Deserialization Failed: \"Fail to decode argument 0 from table0 to null\"', rs/rust_canisters/dfn_core/src/endpoint.rs:33:41")
```

Comment (bitdivine) [reply rs/sns_aggregator/src/upstream.rs:175]: Fixed.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**